torch/distributed/checkpoint/format_utils.py

```diff
@@ -81,3 +76,21 @@ def dcp_to_torch_save(
         no_dist=True,
     )
     torch.save(sd, torch_save_fn)
+
+
+def torch_save_to_dcp(
+    torch_save_fn: Union[str, os.PathLike], dcp_checkpoint_dir: Union[str, os.PathLike]
+):
+    """
+    Given the location of a torch save file, converts it into a DCP checkpoint.
+
+    Args:
+        torch_save_fn: Filename to store the converted Torch save file.
+        dcp_checkpoint_dir: Directory containing the DCP checkpoint.
+
+    .. warning::
+        To avoid OOM, it's recommended to only run this function on a single rank.
+    """
+
+    state_dict = torch.load(torch_save_fn)
+    dcp.save(state_dict, checkpoint_id=dcp_checkpoint_dir, no_dist=True)
```

Comment:
Can we also add main so that users can directly use this file to do conversion? We can do this in a separate PR.